### ISSUE TITLE
Most current version  no longer shows Ckgedit Icon on the side.

### ISSUE THREAD

**Issue by rgleason** [giterlizzi/dokuwiki-template-bootstrap3#447]
### Description

[BUG: ChkEditor is no longer accessible]

### Steps to reproduce

1. [Just install the template]

**Expected behavior:** [What you expected to happen]
I expect to be able to use chkeditor

**Actual behavior:** [What actually happened]

I can't
### Versions
Most current. The date is still wrong.
I have to go back to the previous one once again.
https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn

* [Bootstrap3 Template]
* [DokuWiki]  Most current
* [Plugins] Indexmenu
* [PHP] 
* [Browser]  Firefox

### Screenshots or Logs
[Paste your logs or attach the screenshot]
![Screenshot (324)](https://user-images.githubusercontent.com/2363959/70864007-19c35b00-1f1c-11ea-8b4b-9979b1e7e05f.png)

### Sample page or snippet

[Paste your sample DokuWiki page or snippet using MarkDown code syntax]

**Comment by rgleason** [comment]
Even reinstalling ckgedit does not help.

**Comment by rgleason** [comment]
Revert back to 
https://github.com/giterlizzi/dokuwiki-template-bootstrap3/releases/tag/v2018-02-16
works.

**Comment by giterlizzi** [comment]
I can't see any error or missing feature.
Whats your DokuWiki and CHKEditor version ?

**Comment by rgleason** [comment]
Sorry, I had to revert it back. Its my favorite editor.

**Comment by rgleason** [comment]
The CkgEditor Icon is missing.  There should be two Editor Icons one for DW and one for CkgEdit!
CkgEditor   most current Last updated on  2019-12-09   https://www.dokuwiki.org/plugin:ckgedit
Dokuwiki   Release 2018-04-22b "Greebo"

Since this last report I done what  Myron suggested.
 Myron cannot seem to find anythng wrong either. At his suggestion
Cleaned the cache on several browsers several times. Chrome, Edge, Firefox
Cleaned out our dokuwiki data cache, several times
Reinstalled bootstrap3, then uninstalled it and then installed it.
Also gone back to dokuwiki template.
Also uninstalled chkedit and then installed it. 

Last time this was a problem I went back to I believe it was this
giterlizzi-dokuwiki-template-bootstrap3-v2019-05-22-50-g2a24836.zip
and that fixed the problem.

I will try this again now, but I believe it is a problem with your bootstap3 or something related.

**Comment by rgleason** [comment]
I cannot seem to install that file, it will not upload.
giterlizzi-dokuwiki-template-bootstrap3-v2019-05-22-50-g2a24836.zip

I've changed to Dokuwiki template and removed bootstrap3 completely.
These are the screens  ckgedit still does not show up. If I can get 
giterlizzi-dokuwiki-template-bootstrap3-v2019-05-22-50-g2a24836.zip
installed I think the ckgedit icon will appear.  It is not showing now however with dokuwiki template either!

![Screenshot (407)](https://user-images.githubusercontent.com/2363959/72111893-0a141800-330a-11ea-8e4c-3f8a55b833c9.png)
![Screenshot (408)](https://user-images.githubusercontent.com/2363959/72111896-0da79f00-330a-11ea-9f49-e04b002c9df2.png)

**Comment by rgleason** [comment]
Ok I reinstalled 
https://github.com/giterlizzi/dokuwiki-template-bootstrap3/releases
v2016-04-13
https://github.com/giterlizzi/dokuwiki-template-bootstrap3/archive/v2016-04-13.zip

and now it works with two editor icons, the bottom one being chkedit
So I think you should reopen this,
Your updates to Bootstrap3 and the updates to Dokuwiki Template, both remove chkedit ICON.
from the menu.


![Screenshot (410)](https://user-images.githubusercontent.com/2363959/72112475-b1457f00-330b-11ea-95d0-8147503371ca.png)
![Screenshot (409)](https://user-images.githubusercontent.com/2363959/72112476-b1457f00-330b-11ea-82ba-540a3f2edb8a.png)

**Comment by turnermm** [comment]
I installed the latest bootstrap3 and encountered the same issue as you've reported.  The icons are properly displayed in the previous version but not in the current.  In the last few months I've tested my sectiontoggle plugin on about 30 templates and don't recall seeing this.  Since in most cases these templates had to be newly installed, since they weren't on my system, they were generally the current versions.    One of your images indicates a javascript error:
![image](https://user-images.githubusercontent.com/176447/72127147-583e1100-3334-11ea-890d-f83f55e1353f.png)
But I couldn't reproduce it.

**Comment by rgleason** [comment]
Thank you Myron,
"I installed the latest bootstrap3 and encountered the same issue as you've reported.  The icons are properly displayed in the previous version but not in the current. "  -Good.

"One of your images indicates a javascript error:. [Image of the DW Editor, not ckgedit ]..But I couldn't reproduce it."    I don't see this error, since this is screenshot of DWedit is it consequential?

Is the problem with bootstrap3 or ckgedit?

**Comment by turnermm** [comment]
Then it's possible that you have another plugin that is interfering with either ckgedit or bootstrap or both.  Disable all plugin which are not bundled with Dokuwiki, and clear out your dokuwiki and browser caches

**Comment by rgleason** [comment]
Myron, I can try that, but that's a tall order in some respects, I think.  For example, we really depend on indexmenu, what will happen when remove that?   Will I be able to reestablish that without a lot of trouble with settings etc?

I will start with some of the ones that we aren't using now and whittle.

**Comment by phy25** [comment]
> Myron, I can try that, but that's a tall order in some respects, I think. For example, we really depend on indexmenu, what will happen when remove that? Will I be able to reestablish that without a lot of trouble with settings etc?
> 
> I will start with some of the ones that we aren't using now and whittle.

You can disable them temporarily to see if things goes well and re-enable them immediately.

**Comment by rgleason** [comment]
Thank you phy25, good way to do this.

I have 
**Disabled**
Backup Tool for Dokuwiki
Captcha
Dw2pdf
epub
Gallery Plugin
Info Plugin
Move Plugin
Multiorphan Plugin
Popularity Feedback
Revert Manager
Safefncode plugin
Search Index  Manager
Site Export
Styling Plugin 
Translation Plugin
Dokuwiki Upgrade
Wrap Plugin

**Stilll active**
Plain auth plugin
ACL manager
ckgedit
configuration manager
dwedit
IndexMenu Plugin

When I switch to the current Dokuwiki Plugin, I still have both  Editors DWedit and CkgEdit!
Now I will updated Bootstrap3 and switch to that..

**Comment by rgleason** [comment]
Manually installed  current bootstrap3 template 
https://github.com/LotarProject/dokuwiki-template-bootstrap3/zipball/master

Now there is only ONE editor    DWedit.  We are missing Bootstrap3!!!!!!

Bootstrap3 Template appears to be the culprit!  Can I enable the other plugins now?

**Comment by rgleason** [comment]
Now, when I try Chrome with cache cleared and go back to Dokuwiki Template there is no longer two editor icons.  Just DWedit!   This is  a bit infuriating.

Now I've got indexmenu disabled.  and using either dokuwiki template or bootstrap3 template there is just one editor showing.

**Comment by rgleason** [comment]
MS Edge clear cache, bring up OpenCPN Dokuwiki
https://opencpn.org/wiki/dokuwiki/doku.php?id=opencpn

Just one tool. Reinstall both DWedit and Ckgedit.  on Dokuwiki Template, just one Edit button.
Try bootstrap3  also just one Edit button.

**Comment by rgleason** [comment]
It does not seem to make a difference if all the plugins I can disable are disabled. It still shows on Editor!

Now if I go back to the previous bootstrap3 I willl have both edit buttons.
Well I tried reinstall of bootstrap3 and it did not make a difference.

Then I tried installing  the old bootstrap3
https://github.com/giterlizzi/dokuwiki-template-bootstrap3/archive/v2016-04-13.zip

That does not seem to work anymore just one edit icon shows.   Whoops I had DWedit off.
Turn it on and now there are two edit icons with dokuwiki template

but  I can't seem to get into Admin anymore or log out.   I think the server is confused after doing so much.    There is nothing conclusive here, but  I had turned off everything that could be disaabled and cleared the browser cache..
